src/mame/drivers/kinst.cpp

```diff
@@ -305,6 +307,12 @@ void kinst_state::machine_reset()
 
 	/* set a safe base location for video */
 	m_video_base = &m_rambase[0x30000/4];
+	
+	/* initialize kinst2uk protection registers */
+	prot_sel=0;
+	prot_cnt=0;
+	prot_rega=0;
+	prot_regb=0;
```

Comment:
Is the CPLD actually connected to the machine reset line?  If not, these should only be reset in `machine_start`.